_pages/ato.md

```diff
@@ -38,18 +27,4 @@ While you’re in the ATO process, the following things will occur. These are la
     * A web framework having new backend administrative frameworks
     * The kinds of information you begin to store (e.g. personally identifiable information)
 
-## Tips
-
-One thing that will help the process is great documentation, which can mitigate some problems from occurring during the ATO process. Documentation, and specifically your README, should reflect a high level narrative of the architecture and data flows of the application. Questions to consider include:
-
-* What does it do?
-* How does it move information around?
-* What does it accomplish by doing it?
-
-## Optional - 90-day Authority to Test
-
-* 18F offers a 90 day authority to test security clearance process.
-    * Estimated to take approximately 2 weeks to get materials together and information security team has a service level agreement of 2 weeks to approve a completed system security plan, do the testing, and give a 90 day authorization.
-    * The 90 day ATO can be re-upped as long as the security boundary hasn’t changed; the automatic scans must not show any new vulnerabilities.
-* [Sensitive PII](../security/pii) is documented here; this process is not analogous to the PIA/SORN process and they must be handled separately
```

Comment:
Is this PII reference here part of the Authority To Test section? It didn't make it over into the new page, but I can't tell if it should really be part of this section.